product/models.py

```diff
@@ -0,0 +1,46 @@
+from django.db   import models
+
+#from user.models import User,Order
+
+# Create your models here.
+
+class ShopCategory(models.Model):
+    name = models.CharField(max_length=50)
+
+    class Meta:
+        db_table = 'shop_categories'
+
+class Product(models.Model):
+    category    = models.ForeignKey(ShopCategory, on_delete=models.CASCADE)
+    name        = models.CharField(max_length=50)
+    price       = models.IntegerField(default=0)
+    description = models.TextField()
+    shipping    = models.TextField()
+    orders      = models.ManyToManyField('user.Order',through='Cart', related_name='orders')
```

Comment:
Order model이 user app에 존재하나요??
```suggestion
    orders      = models.ManyToManyField('order.Order',through='Cart', related_name='orders')
```
Order app을 따로 만들어서 분리해주세요~

Reply:
넵